### DISCUSSION TITLE
notes/json/

### DISCUSSION THREAD

**Discussion by giscus-x** [xaoxuu/blog-comments#50] [Announcements]
# notes/json/

https://xaoxuu.com/notes/json/

**Comment by Alanhays** [comment]
你好，请问这种工具页面是如何做到的？

**Reply by xaoxuu** [reply]
内嵌iframe

**Comment by x-liao** [comment]
点击导航图标跳转到/notes/路径时会出现全屏封面，如何设置不显示那个'开始使用'，直接显示内容

**Reply by x-liao** [reply]
找到设置项coverpage了

**Comment by xlenco** [comment]
佬你这个页面开源吗？我想看看咋配置的

**Reply by wsjz** [reply]
我感觉文档已经跟不上了，有点想放弃

**Reply by xaoxuu** [reply]
源码： https://github.com/xaoxuu/hexo-theme-stellar-examples

**Comment by wsjz** [comment]
这是来自QQ邮箱的假期自动回复邮件。
